semantic/tasks/rtl/build.js

```diff
@@ -0,0 +1,132 @@
+/*******************************
+          Build Task
+*******************************/
+
+var
+  gulp         = require('gulp'),
+
+  // node dependencies
+  fs           = require('fs'),
+
+  // gulp dependencies
+  autoprefixer = require('gulp-autoprefixer'),
+  chmod        = require('gulp-chmod'),
+  clone        = require('gulp-clone'),
+  flatten      = require('gulp-flatten'),
+  gulpif       = require('gulp-if'),
+  less         = require('gulp-less'),
+  minifyCSS    = require('gulp-clean-css'),
+  plumber      = require('gulp-plumber'),
+  print        = require('gulp-print').default,
+  rename       = require('gulp-rename'),
+  replace      = require('gulp-replace'),
+  rtlcss       = require('gulp-rtlcss'),
+  uglify       = require('gulp-uglify'),
+
+  // user config
+  config       = require('../config/user'),
+
+  // install config
+  tasks        = require('../config/tasks'),
+  install      = require('../config/project/install'),
+
+  // shorthand
+  globs        = config.globs,
+  assets       = config.paths.assets,
+  output       = config.paths.output,
+  source       = config.paths.source,
+
+  banner       = tasks.banner,
+  comments     = tasks.regExp.comments,
+  log          = tasks.log,
+  settings     = tasks.settings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)